product_abc_classification_sale_stock/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     "version": "16.0.1.0.2",
     "license": "AGPL-3",
     "website": "https://github.com/OCA/product-attribute",
-    "maintainers": ["rousseldenis", "lmignon", "lmarion-source"],
+    "maintainers": ["rousseldenis", "lmignon", "lmarion-source", "kobros-tech"],
```

Comment:
You could add your name into the contributors but not into the authors since your changes are related to some little issue, not to a large change, improvement, new functionality.